### DISCUSSION TITLE
Repo Assignment

### DISCUSSION THREAD

**Discussion by jpmsrh** [community/community#45429] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

What's your thoughts on archiving verses deleting when cloning Repos?

**Comment by jge162** [comment]
Hello @jpmsrh whether to archive or delete the original repository depends on the specific needs of your project and the nature of the repository. Archiving the original repository may be the best option if you want to keep the history and code for reference, while deleting it may be the best option if you want to start fresh with a new development effort.
